eng/versioning/version_client.txt

```diff
@@ -3,12 +3,12 @@
 
 com.azure:azure-sdk-parent;1.5.0;1.5.0
 com.azure:azure-client-sdk-parent;1.6.0;1.6.0
-com.azure:azure-core;1.0.0-preview.7;1.0.0-preview.7
-com.azure:azure-core-amqp;1.0.0-preview.7;1.0.0-preview.7
-com.azure:azure-core-http-netty;1.0.0-preview.7;1.0.0-preview.7
-com.azure:azure-core-http-okhttp;1.0.0-preview.7;1.0.0-preview.7
-com.azure:azure-core-management;1.0.0-preview.7;1.0.0-preview.7
-com.azure:azure-core-test;1.0.0-preview.7;1.0.0-preview.7
+com.azure:azure-core;1.0.0;1.0.0
+com.azure:azure-core-amqp;1.0.0;1.0.0
+com.azure:azure-core-http-netty;1.0.0;1.0.0
+com.azure:azure-core-http-okhttp;1.0.0;1.0.0
+com.azure:azure-core-management;1.0.0-preview.5;1.0.0-preview.5
```

Comment:
Did you run @JimSuplizio 's script to update these version numbers? Is it intentional to downgrade this to 5?

Reply:
I looked up maven central - the latest published was preview.4

https://search.maven.org/search?q=g:com.azure%20AND%20a:azure-core-management&core=gav

Looks like this was an error in the repo. Happy to make it preview.7 but that means we've skipped preview numbers in Maven Central...

Reply:
I'm going to revert back to preview.7, since that is what it was set at. It looks strange after we GA azure-core because we won't have any/many other preview.7 libraries so it looks like we skipped instead of just introduced.